zerver/views/typing.py

```diff
@@ -12,22 +17,21 @@
 from zerver.models import UserProfile
 
 VALID_OPERATOR_TYPES = ["start", "stop"]
-VALID_RECIPIENT_TYPES = ["direct", "stream"]
+VALID_RECIPIENT_TYPES = ["direct", "stream", "message_edit", "direct_message_edit"]
 
 
 @has_request_variables
 def send_notification_backend(
     request: HttpRequest,
     user_profile: UserProfile,
-    req_type: str = REQ(
-        "type", str_validator=check_string_in(VALID_RECIPIENT_TYPES), default="direct"
-    ),
+    req_type: str = REQ("type", str_validator=check_string_in(VALID_RECIPIENT_TYPES)),
     operator: str = REQ("op", str_validator=check_string_in(VALID_OPERATOR_TYPES)),
     notification_to: Optional[List[int]] = REQ(
         "to", json_validator=check_list(check_int), default=None
     ),
     stream_id: Optional[int] = REQ(json_validator=check_int, default=None),
     topic: Optional[str] = REQ("topic", default=None),
+    message_id: Optional[int] = REQ(json_validator=check_int, default=None),
```

Comment:
I started an API design thread for some further details of this API design worth discussing as a group.

https://chat.zulip.org/#narrow/stream/378-api-design/topic/typing.20notifications.20for.20editing.20messages/near/1720430

Reply:
added it to PR description.